### PR TITLE
Bump to 1.66.0

### PR DESCRIPTION
**Pull request by tschoonj**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,14 +9,12 @@ package:
 
 source:
   url: https://download.gnome.org/sources/{{ name }}/{{ '.'.join(version.split('.')[:2]) }}/{{ name }}-{{ version }}.tar.xz
-  sha256: 80beae6728c134521926affff9b2e97125749b38d38744dc901f4010ee3e7fa7
+  sha256: 3d66ea3aec7c3f8f1f83d89db3e78e18e36adc22b8bba45298119f0b3ea50060
   patches:
-    - meson-rpaths.patch  # [linux]
-    - gcc7-rpath-link.patch  # [linux]
```
Comment (ocefpaf) [inline recipe/meta.yaml:15]: Did these get upstream or they are not applying cleanly and need a refresh?

Comment (tschoonj) [reply recipe/meta.yaml:15]: I don't know to be honest. But it appears that they are no longer necessary as the tests run fine without them.

Comment (ocefpaf) [reply recipe/meta.yaml:15]: Tests may be misleading, they may not cover that part. I'll defer to @pkgw to merge this one b/c I don't really know enough about `gobject-introspection` to do it myself.

File: recipe/yum_requirements.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+libX11
+libXcomposite
+libXcursor
+libXdamage
+libXext
+libXfixes
+libXi
+libXinerama
+libXrandr
```
Comment (scopatz) [inline recipe/yum_requirements.txt:9]: Why was this file added?

Comment (tschoonj) [reply recipe/yum_requirements.txt:9]: I added `pygobject` as a downstream test, since it is the only recipe that has a run-time dependency on `gobject-introspection`, and problems with this recipe are most likely to show up there.

Since the `pygobject` tests depend on `gtk3`, I need to pull in these yum dependencies.

File: recipe/meta.yaml (hunk 1 above)
Comment (pkgw) [reply recipe/meta.yaml:15]: The meson-rpaths patch should be unnecessary now that we're patching Meson (sigh). I'd be less sure about `gcc7-rpath-link`, but the comment in the patch explicitly says that building pygobject tests whether it works, and we're doing that now, so I think that issue must have been fixed upstream.
